getting-started-tutorial/01-mlrun-basics.ipynb

```diff
@@ -292,9 +313,9 @@
     "\n",
     "Projects are visible in the MLRun dashboard only after they're saved to the MLRun database, which happens whenever you run code for a project.\n",
     "\n",
-    "The following code creates a project named \"getting-started-iris-&lt;V3IO_USERNAME&gt;\", where **&lt;V3IO_USERNAME&gt;** is your current running username in the platform, and sets the project directory to a **conf** directory in the current tutorial directory (**/User/getting-started-tutorial/conf**).\n",
+    "The following code creates a project named \"getting-started-iris-&lt;V3IO_USERNAME&gt;\", where **&lt;V3IO_USERNAME&gt;** is the username of the current running platform user, and sets the project directory to a **conf** directory in the current tutorial directory (**/User/getting-started-tutorial/conf**).\n",
```

Comment:
need to replace with `<username>` (not the env var..)

Reply:
@yaronha I didn't write this doc and I don't know what you want me to replace the user-name reference with.
Also, if I understand correctly, you want to remove this entire cell, which deals with projects (no?), which if confirmed with Product would render this discussion moot.